gramineproject/gramine
# [Docs] Add remote attestation terminology

## Description of the changes 

In particular, the terminology follows TCG DICE Attestation.

Extracted from #1039.

## How to test this PR? 

Check the RTD pages:
- https://gramine--1927.org.readthedocs.build/en/1927/glossary.html
- https://gramine--1927.org.readthedocs.build/en/1927/sgx-intro.html


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gramine/1927)
